src/app/pages/datasets/components/dataset-details-card/dataset-details-card.component.html

```diff
@@ -53,13 +53,22 @@ <h3 mat-card-title *ngIf="isZvol">{{ 'Zvol Details' | translate }}</h3>
         </span>
       </div>
       <div class="details-item">
-        <span class="label">{{ 'Path:' | translate }} </span>
+        <span class="label">{{ 'Path' | translate }}:</span>
         <span *ngIf="!isLoading"
           [matTooltip]="dataset.name"
           matTooltipPosition="above"
           class="value dataset-path">
           {{ dataset.name }}
         </span>
+        <ix-copy-btn [text]="dataset.name"></ix-copy-btn>
+      </div>
+      <div class="details-item">
+        <span class="label">{{ 'Comments' | translate }}:</span>
+        <span *ngIf="!isLoading" class="value dataset-comments">
```

Comment:
1. Let's not show the row when comment is not available.
2. Long comment won't wrap.
3. It's weird, but adding a comment on parent will show the same comment on all children. I'm not sure if this is backend or not, but I don't think we should show parent comments like this.

Reply:
![image](https://user-images.githubusercontent.com/351613/187211591-a53200fd-c7b0-4f18-aee2-b224356bbdd7.png)

1. Ok done
2. Fixed. If the text is greater than 128 symbols, it will be truncated, and copy button will be added at the end.
3. Good catch. The comments are a ZFS property and can be inherited. Added check for the source of comments